### PR TITLE
Make the cursor no longer blinking when move, as same as the effect of iOS platform.

### LINKED ISSUE #105454: Long press blank and drag it to move cursor. The cursor shouldn't blink.
## Steps to Reproduce

The blinking animation of Flutter app looks different with iOS native app when I move the cursor.

There're screen records of Flutter app and iOS native app.

### Flutter

https://user-images.githubusercontent.com/7621572/172211845-a89a9dda-f60a-4c9a-aec3-9e96c1a53409.mov

### iOS Native
https://user-images.githubusercontent.com/7621572/172211895-da8897f1-66b3-4109-b296-e0b709f07a65.mov

### Steps

1. Long press and drag the blank bar.
2. Look at the cursor. The cursor will keep blinking while dragging. But iOS won't.

**flutter doctor**
<details>
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel unknown, 3.0.1, on macOS 12.4 21F79 darwin-x64, locale
    zh-Hans-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.4)
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.67.2)
[✓] Connected device (1 available)
[✓] HTTP Host Availability

• No issues found!
```
</details>

### PR DESCRIPTION
**Pull request by talisk**
Fix #105454 

**Before**

https://user-images.githubusercontent.com/7621572/172214290-893f0868-0138-4c30-b8d0-ed02fada9331.mov

**After**

https://user-images.githubusercontent.com/7621572/172214298-f6a53820-1d65-4162-a21f-eae295c67342.mov

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 1]
```diff
@@ -2097,6 +2097,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
           _floatingCursorResetController!.stop();
           _onFloatingCursorResetTick();
         }
+        // Stop cursor blinking and making it visable
```
Comment (justinmc) [inline packages/flutter/lib/src/widgets/editable_text.dart:2100]: Period at the end of the sentence here.

File: packages/flutter/test/widgets/editable_text_cursor_test.dart [hunk 2]
```diff
@@ -813,13 +806,6 @@ void main() {
     );
 
     expect(find.byType(EditableText), paints
-      ..rrect(
```
Comment (justinmc) [inline packages/flutter/test/widgets/editable_text_cursor_test.dart:816]: Can you explain why these two  `..rrect`s had to be removed?

**Review by justinmc — COMMENTED** [review]
Thanks for fixing this!  The solution looks good, but I want to make sure this is tested properly.  I have a question below about where you removed `..rect` in the tests.  If there was a regression and the cursor was made to blink again, would those tests fail or would they still pass?

File: packages/flutter/lib/src/widgets/editable_text.dart (hunk 1 above)
Comment (talisk) [reply packages/flutter/lib/src/widgets/editable_text.dart]: Done.

File: packages/flutter/test/widgets/editable_text_cursor_test.dart (hunk 2 above)
Comment (talisk) [reply packages/flutter/test/widgets/editable_text_cursor_test.dart:816]: In fact, this PR will fix two differences. 

First, the blue(Color 0xbf2196f3) cursor no longer blinks while dragging. Second, when dragging, the gray(Color 0xff999999) cursor that moves between text is also removed. These are the differences between the Flutter and the iOS native.
These two `..rrect`s validate the previous grey(Color 0xff999999) cursor at the bottom. 

image

Comment (justinmc) [reply packages/flutter/test/widgets/editable_text_cursor_test.dart:816]: Got it, thanks for the clear explanation with screenshots.

**Comment by talisk** [comment]
OK, I will write test case soon.

**Comment by goderbauer** [comment]
@talisk It looks like some of the checks are failing. Can you check whether this is related to your change and fix the failures if necessary?

**Comment by talisk** [comment]
@goderbauer @justinmc 
I opened a new PR #107221 and it's pipeline passed. Shall we close this one(#105455 ) and merge the new one(#107221 )?

**Comment by LongCatIsLooong** [comment]
@talisk should I close this pull request in favor of https://github.com/flutter/flutter/pull/107221?

**Pull request closed without merging**